code/src/NotFound.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import "./pages/notFound.css"
+
+
+export const NotFound = () => {
+    return (
+        <div className="not-found">
+            <h4>Oh no, we can't find that movie.</h4>
+            <img src="https://media.giphy.com/media/l2JehQ2GitHGdVG9y/source.gif" alt="homer simpson lost" />
+            <h3>Why don't you go back and check out our movies list, a perfect inspiration for a cozy night!</h3>
+            <Link to="/">
+                <p className="error-message">Go back home</p>
+            </Link>
+        </div>
+    );
```

Comment:
This doesn’t show for me? This is what I see when I go to a movie that doesn’t exist. 🤔
![image](https://user-images.githubusercontent.com/65863845/98558956-56bbdd00-22a6-11eb-9e25-a51a972e6ba7.png)

Reply:
OK, so Homer shows here: 
https://project-movie-releases-kat.netlify.app/404

but not here: 
https://project-movie-releases-kat.netlify.app/movies/somemovie